.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,3 +6,5 @@ contact_links:
   - name: "💬 IRC: #pypa"
     url: https://kiwiirc.com/nextclient/#ircs://irc.libera.chat:+6697/pypa
     about: Chat with devs
+  - name: "(maintainers only) Blank issue"
+    url: https://github.com/pypa/pip/issues/new 
```

Comment:
Seems like this line has an additional whitespace at the end. You only have to remove it.

Reply:
_sigh_